packages/AutoTDD-Core.package/ATDDGitHubActionsParser.class/instance/primitiveParseLatestWorkflowRunConclusion..st

```diff
@@ -1,4 +1,4 @@
 primitive-parsing
 primitiveParseLatestWorkflowRunConclusion: aJson
 	
-	^ aJson at: 'conclusion' ifAbsent: [ 'unknown' ]
\ No newline at end of file
+	^ (aJson at: 'conclusion') ifNil: [ 'unknown' ]
```

Comment:
```suggestion
	^ (aJson at: 'conclusion' ifAbsent: [ 'unknown' ] ) ifNil: [ 'unknown' ]
```

Reply:
Could be absent (unless squeak returns nil by default when accessing an absent entry)

Reply:
It does :)

<img width="1106" alt="Screenshot 2021-05-23 at 21 40 02" src="https://user-images.githubusercontent.com/58258541/119274308-895c7000-bc0f-11eb-8000-1675150970fc.png">